cns/restserver/restserver.go

```diff
@@ -906,12 +906,14 @@ func (service *httpRestService) createOrUpdateNetworkContainer(w http.ResponseWr
 
 	switch r.Method {
 	case "POST":
-		nc := service.networkContainer
-		err := nc.Create(req)
-		if err != nil {
-			returnMessage = fmt.Sprintf("[Azure CNS] Error. CreateOrUpdateNetworkContainer failed %v", err.Error())
-			returnCode = UnexpectedError
-			break
+		if req.NetworkContainerType == cns.WebApps {
+			nc := service.networkContainer
+			err := nc.Create(req)
```

Comment:
Let's do this to keep our code consistent
if err := nc.Create(req); err != nil {
}